branching.md

```
@@ -1 +1,3 @@
 #This is the branching strategy you need for git.
```

Markdownlint (0.37.0)

No space after hash on atx style heading

(MD018, no-missing-space-atx)